chatgpt.sh

```diff
@@ -259,7 +259,11 @@ while $running; do
 			echo "Would you like to open it? (Yes/No)"
 			read -e answer
 			if [ "$answer" == "Yes" ] || [ "$answer" == "yes" ] || [ "$answer" == "y" ] || [ "$answer" == "Y" ] || [ "$answer" == "ok" ]; then
-				open "${image_url}"
+				if [[ -z "${CHATGPT_PLUMBER}" ]]; then
```

Comment:
What is the value of the variable CHATGPT_PLUMBER ?
If I merged this PR as it is, the script wouldn't work. Please consider suggesting changes that you have tested locally first.

Reply:
Hello @0xacx, I tested the code before submitting the PR, and it worked properly. However, I'm unsure why the script is currently not working. It's clear from the patch that the code checks the environment and uses `open` as the default value if `CHATGPT_PLUMBER` is not set.